### PR TITLE
Adding a notice for the next event. TEMPORARY!

### PR DESCRIPTION
**Pull request by monsieurhannes**

### CODE REVIEW

**Comment by monsieurhannes** [comment]
We have to remember to remove it on Sunday, but I think it's good to have it :)

File: index.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ We are a loose community of people that wants to help reducing the amount of ele
 
 See below for the upcoming planned meetups.
 
+**Notice:**
+Next Repair Café on August 28th is at [Garaget Library](https://malmo.se/Uppleva-och-gora/Biblioteken/Vara-bibliotek/Garaget.html)
```
Comment (qerub) [inline index.md]: August 28th? That's today! It says September 1th in the calendar.

Comment (monsieurhannes) [reply index.md]: Oh, yes, good point :D

File: index.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ We are a loose community of people that wants to help reducing the amount of ele
...
+<iframe width="100%" height="350" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="https://www.openstreetmap.org/export/embed.html?bbox=13.015057146549227%2C55.5842316196598%2C13.019536435604097%2C55.58564599288903&amp;layer=mapnik&amp;marker=55.58493881264474%2C13.01729679107666" style="border: 1px solid black"></iframe><br/><small><a href="https://www.openstreetmap.org/?mlat=55.58494&amp;mlon=13.01730#map=19/55.58494/13.01730">View Larger Map</a></small>
```
Comment (qerub) [inline index.md]: Can you make the initial view zoomed in? Seeing that the event is on planet Tellus is not very helpful.

Comment (monsieurhannes) [reply index.md]: What do you mean? It's zoomed to the location, isn't it?

Comment (qerub) [reply index.md]: This is what it looks like to me:

image

Comment (qerub) [reply index.md]: Hmm... Sometimes when I load/refresh the page it's zoomed in.

Comment (qerub) [reply index.md]: Could be some strange CSS interaction: https://help.openstreetmap.org/questions/20335/embedded-html-displays-zoomed-out

Comment (qerub) [reply index.md]: ...so let's not worry about it.

Comment (monsieurhannes) [reply index.md]: Strange, but could it be that you have 1001 plugins installed that block everything? :D 
bild

Comment (monsieurhannes) [reply index.md]: Should we merge or is there a fix for the CSS-issue?

Comment (qerub) [reply index.md]: I say we merge.

**Pull request merged**